cli/util/util_test.go

```diff
@@ -63,3 +66,47 @@ func TestUtilOps(t *testing.T) {
 	e.Run(t, "neo-go", "util", "ops", "--hex", "--in", tmp) // hex from file
 	check(t)
 }
+func TestUtilCancelTx(t *testing.T) {
+	e := testcli.NewExecutorSuspended(t)
+
+	w, err := wallet.NewWalletFromFile("../testdata/testwallet.json")
+	require.NoError(t, err)
+
+	tmp := t.TempDir()
+	configPath := filepath.Join(tmp, "config.yaml")
+	cfg := config.Wallet{
+		Path:     testcli.ValidatorWallet,
+		Password: testcli.ValidatorPass,
+	}
+	yml, err := yaml.Marshal(cfg)
+	require.NoError(t, err)
+	require.NoError(t, os.WriteFile(configPath, yml, 0666))
+
+	transferArgs := []string{
+		"neo-go", "wallet", "nep17", "transfer",
+		"--rpc-endpoint", "http://" + e.RPC.Addresses()[0],
+		"--wallet", testcli.ValidatorWallet,
+		"--to", w.Accounts[0].Address,
+		"--token", "NEO",
+		"--from", testcli.ValidatorAddr,
+		"--force",
+	}
+
+	e.In.WriteString("one\r")
+	e.Run(t, append(transferArgs, "--amount", "1")...)
+	line := e.GetNextLine(t)
+	txHash, err := util.Uint256DecodeStringLE(line)
+	require.NoError(t, err)
+
+	_, ok := e.Chain.GetMemPool().TryGetValue(txHash)
+	require.True(t, ok)
+
+	e.Run(t, "neo-go", "util", "canceltx", txHash.StringLE(), "-r", "http://"+e.RPC.Addresses()[0], "--wallet-config", configPath, "--address", testcli.ValidatorAddr)
```

Comment:
You don't need a wallet config file, just add password before this line: `e.In.WriteString("one\r")`, and replace `--wallet-config` with `--wallet`.